src/main_app/db/create_helper.py

```
@@ -43,20 +43,19 @@ def create_views(_db: SQLAlchemy) -> None:
                     try:
                         with conn.begin():
                             conn.execute(text(f"DROP VIEW IF EXISTS {table.name}"))
-                    except Exception:
-                        logger.exception("Failed to drop view %s", table.name)
+                    except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)